docs/orchestration/execution/overview.md

```diff
@@ -13,7 +12,7 @@ Executing flows using the Prefect API is accomplished through two powerful abstr
 
 ## Storage
 
-[Storage](https://docs.prefect.io/api/latest/storage.html) objects are pieces of functionality which define how and where a Flow should be stored. Prefect supports storage options ranging from ephemeral in-memory storage to Docker images which can be stored in registries.
+[Storage](/api/latest/storage.html) objects are pieces of functionality which define how and where a Flow should be stored. Prefect supports storage options ranging from ephemeral in-memory storage to Docker images which can be stored in registries.
```

Comment:
I know you were just updating the URL, but this paragraph needs some attention. I don't think we have in memory storage? And "pieces of functionality" is awkward.

Reply:
I've completely rewritten this with proper link to supported storage types.